ee/tabby-schema/src/schema/auth.rs

```diff
@@ -348,6 +370,75 @@ pub struct UpdateOAuthCredentialInput {
     pub client_secret: Option<String>,
 }
 
+#[derive(GraphQLEnum)]
+pub enum LdapEncryptionKind {
+    None,
+    StartTLS,
+    LDAPS,
+}
+
+#[derive(GraphQLInputObject, Validate)]
+pub struct UpdateLdapCredentialInput {
+    #[validate(length(
+        min = 1,
+        code = "host",
+        message = "host should not be empty and should be a valid hostname or IP address"
+    ))]
+    pub host: String,
+    pub port: i32,
+
+    #[validate(length(min = 1, code = "bind_dn", message = "bind_dn cannot be empty"))]
+    pub bind_dn: String,
+    #[validate(length(
+        min = 1,
+        code = "bind_password",
+        message = "bind_password cannot be empty"
+    ))]
+    pub bind_password: String,
+
+    #[validate(length(min = 1, code = "base_dn", message = "base_dn cannot be empty"))]
+    pub base_dn: String,
+    #[validate(length(
+        min = 1,
+        code = "user_filter",
+        message = "user_filter cannot be empty, and should be in the format of `(uid=%s)`"
+    ))]
+    pub user_filter: String,
+
+    pub encryption: LdapEncryptionKind,
+    pub skip_tls_verify: bool,
+
+    #[validate(length(
+        min = 1,
+        code = "email_attribute",
+        message = "email_attribute cannot be empty"
+    ))]
+    pub email_attribute: String,
+    #[validate(length(
+        min = 1,
+        code = "name_attribute",
+        message = "name_attribute cannot be empty"
+    ))]
+    pub name_attribute: String,
+}
+
+#[derive(GraphQLObject)]
+pub struct LdapCredential {
+    pub host: String,
+    pub port: i32,
+    pub bind_dn: String,
+    pub bind_password: String,
```

Comment:
```suggestion
    #[graphql(skip)]
    pub bind_password: String,
```

or consider remove the field directly.